shuffle/move/examples/main/devapi.ts

```diff
@@ -10,29 +10,48 @@
 // deno-lint-ignore-file no-explicit-any
 // deno-lint-ignore-file ban-types
 import * as context from "./context.ts";
+import { delay } from "https://deno.land/std@0.114.0/async/delay.ts";
 
-console.log(await ledgerInfo());
-
+// Retrieves the ledger information as defined by the root /
+// of the Developer API
 export async function ledgerInfo() {
   return await checkingFetch(context.relativeUrl("/"));
 }
 
+// Returns a list of transactions, ascending from page 0, or, if passed
+// a version or hash, returns a specific transaction.
 export async function transactions(versionOrHash?: string) {
   if (versionOrHash) {
     return await checkingFetch(
       context.relativeUrl(`/transactions/${versionOrHash}`),
     );
   }
+  // TODO: Have below return a list of transactions desc by sequence number
   return await checkingFetch(context.relativeUrl("/transactions"));
 }
 
+// Polls for a specific transaction to complete, returning the `success`
+// field when no longer pending.
+export async function transactionSuccess(versionOrHash: string): Promise<boolean> {
+  for (let i = 0; i < 4; i++) {
```

Comment:
probably increase this count from 4 to 30, in local env it's fine and fast, in production env (like premainnet), execute txn maybe up to couple seconds.